### PR TITLE
Smashing Magazine Heatmap

### PR DESCRIPTION
**Pull request by anewman15**
In this PR, we created a heatmap of the Smashing Magazine website.

It consists of various sections including a navbar, a main contents area, latest articles section, some banner sections, a section with links to different guides and a footer area.

We mainly worked on converting the background and text colors to grayscale and examine on how the website looks and feels under varying shades of grey. We also tested how font size and weight plays a role in determining visual hierarchy.

### CODE REVIEW

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @anewman15 & @ElijahTobs  :wave:
-
You have done a splendid job! :tada: The project looks great. Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
## [REQUIRED] - FIX -The README.
The project's README's live link should be updated. The description is well documented, and also in the social links of @anewman15, let's replace the placeholders with the actual social links.
Please makes changes and submit another review request.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by anewman15** [comment]
Thanks @christianotieno, we have made the changes

**Comment by uryelah** [comment]
Hello **Abdullah Numan & Elijah Ayandokun**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**First of all good job fixing the readme file, how you present your projects will be very important for recruiters ;-) 🌟**

There are still a couple of requirements you'll need to implement in order to go to the next project. So keep calm and write markup 💪

---------     

## Required Changes 🏗 👷‍♀ 

- [x] The navbar should position elements as they are in the original page. You could use divs to represent each nav icon, and remember to also add the text.

- [x] Each article in the page should actually use the `<article>` tag, let's remember semantic html. So for the elements with the `.article` and similar classes make then articles, not divs.

- [x] Remember to add the text and divs representing the images for each article. Text represents a lot of visual weight after all. 

-----

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by anewman15** [comment]
Thanks @uryelah. We have made the suggested changes

File: index.html [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+        <i class="fas fa-search"></i>&nbsp;&nbsp;&nbsp;<span>Topics</span>
+      </div>
+    </div>
+
+  </nav>
+
+  <main class="main-area">
+    <div class="main-articles">
+      <div class="center"></div>
+      <div class="article1"></div>
```
Comment (uryelah) [inline index.html]: - [x] These, .article1, 2, 3, etc. Should be `<articles>`
- [x] Remember to also add text to them.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="latest-link">
+      <a class="links-a" href="#">SEE ALL LATEST ARTICLES <i class="fas fa-long-arrow-alt-right"></i></a>
+    </div>
+
+    <div class="design-handbook">
```
Comment (uryelah) [inline index.html]: This div and other ones are still missing some visual content. You don't need to literally copy the illustration and make it grey. You can just put the missing text and divs to represent any illustration or picture.

**Review by uryelah — CHANGES_REQUESTED** [review]
Good job improving the project since the last review, you still need to change some of the divs to articles. An article in HTMl is a block of content that makes sense on its own, [you can read more here if you want a refresher.](https://www.w3schools.com/html/html5_semantic_elements.asp)

---------     

## Required Changes 🏗 👷‍♀

**Comment by anewman15** [comment]
Done @uryelah. I hope this is enough...

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved 🏆 🎈 

Hi **TEAM**, you did a great job fixing all the recommended changes pointed out by the previous TSE's 👍 congratulations. The project looks great and I hereby approve it 😃 
However, before we merge to master, let us apply the DRY rule to our CSS file. 🙂  Below is a image indicating part of the repeating code. Kindly look through them and make the fixes
DP
Kindly update the README file with the right live link and add email to the author section

Good luck with your next projects 😄 
_Happy coding_ 👨‍💻 
Daniel Larbi Addo on slack

**Comment by ElijahTobs** [comment]
@addod19 The DRY rule has been applied to our CSS file

**Pull request merged**